graylog2-web-interface/src/integrations/aws/cloudwatch/FormAdvancedOptions.tsx

```diff
@@ -54,6 +54,15 @@ const FormAdvancedOptions = ({ onChange }: FormAdvancedOptionsProps) => {
         help='Add field with the Flow Log prefix e. g. "src_addr" -> "flow_log_src_addr".'
       />
 
+      <Input
+        id="overrideSource"
+        type="text"
+        value={overrideSource?.value}
+        onChange={onChange}
+        label="Override Source (optional)"
+        help="The source is set to the Kinesis message by default. Set this if you want to override it with a custom value."
```

Comment:
Similar to the comment on `graylog2-server/src/main/java/org/graylog/integrations/aws/inputs/AWSInput.java`, `"aws-kinesis-raw-logs"` is the current default value.